config.py

```diff
@@ -9,6 +9,8 @@ class BaseConfig:
     DOMAIN_URL = os.getenv("DOMAIN_URL")
     DOMAIN_URL_PROTOCOL = os.getenv("DOMAIN_URL_PROTOCOL")
 
+    SECRET_KEY = os.getenv("SECRET_KEY")
+
```

Comment:
```suggestion
SESSION_COOKIE_SECURE = os.getenv("SESSION_COOKIE_SECURE")
```
It's set in the k8s manifests, but not actually used anywhere in the code. 
Ensures that the request cookies are sent only when using HTTPS. 
The default is False, so we want to set it to True as in the manifests.